app/car.py

```diff
@@ -0,0 +1,47 @@
+import math
+from app.shop import Shop
+from app.customer import Customer
+
+
+class PriсeKm:
+    def __init__(self, customer: Customer) -> list:
+        self.location_customer1 = customer.location_customer1
+        self.fuel_consumption_car3 = customer.fuel_consumption_car3
+        self.priсe_fuel = customer.priсe_fuel
+
+    def distance_priсe() -> dict:
```

Comment:
try to separate this code to a few functions with different responsibilities, it is hard to read